### DISCUSSION TITLE
personal-development-farvardin-1402/

### DISCUSSION THREAD

**Discussion by giscus** [amir-pourmand/hugo-papermod-farsi-template#12] [Announcements]
# personal-development-farvardin-1402/

گزارش توسعه مهارتی من در فروردین ۱۴۰۲.

https://aprd.ir/personal-development-farvardin-1402/

**Comment by HashemAlavi** [comment]
سلام آقای پورمند. من هاشم علوی‌ام. خیلی گزارش خوبی بود. اتفاقاًٌ خیلی هم منظم بود (در مقابل چیزی که من نوشتم😂)

ولی این گزارش نوشتن‌ها خیلی چیز خوب و جالبیه. حس خوبی می‌گیرم. هم از نوشتنش و هم از خوندنش:)

بگذریم.

راستش بیشتر بخاطر دیدن لینک وبلاگ خودم ترغیب شدم که اینجا کامنت بذارم. هم تعجب کردم و هم خیلی خوشحال شدم. و البته بیشتر تو این فکرم که چجوری به این چیزایی که می‌خونم و می‌نویسم قراره عمل کنم.:/
هر لحظه هم تو این فکرم که نکنه به ماشین یادگیری بدون عمل تبدیل بشم.

در مورد فیلم برادران لیلا هم یک اتفاق جالب افتاده برام. تقریباً هر جا میرم یکی بهم می‌گه بشین این فیلم رو ببین. فکر کنم این شیشم یا هفتیمن باریه که اینجوری می‌شه:)

یک نکته دیگه هم در مورد همین تعادل در زندگی بگم و بحثم رو تموم کنم. این مفهوم به قدری بین همه جا افتاده و همه چشم بسته قبولش دارن که تقریباً نمی‌شه به راحتی کسی رو پیدا کرد که کمی به این تعادل در زندگی مشکوک باشه. همه می‌خوان برای خودشون (و حتی بقیه) یک زندگی متعادل بسازن. همه چی مساوی. گرفتن زندگی خودشون رو تیکه تیکه کردن و مثلاً یک تایمی برا کار و یک تایمی هم برا مطالعه و یک تایمی هم برا تفریح و ... و جالب هم اینه که حواسشون هست از هیچ بخشی دریغ نکنن و تا جای ممکن زمانشون رو عادلانه بین این کارها تقسیم کنن (بعضیا هم سعی می‌کنن مساوی باشه این تقسیم‌بندی). بگذریم. انگار نمی‌شه کاریش کرد.

بازم ممنون که می‌نویسی و ممنون که این کامنت رو می‌خونی:) آرامش و رضایت رو برات آرزو می‌کنم.

**Reply by pourmand1376** [reply]
سلام آقا هاشم. 

خوشحالم که مطلبم برات جالب بوده. وقتی به گذشته نگاه می‌کنم واقعا یادم نیست ۵ سال پیش روزهام رو چطوری سپری می‌کردم. این یک سند زنده و البته عمومی هست از گذشته. شاید نوشتن این گزارش‌ها بصورت عمومی بتونه به تقویت عزت نفس هم کمک کنه. 

در مورد فیلم بگم که من بایاس ذهنی زیادی نسبت به فیلم‌های ایرانی دارم. تقریبا از اکثرشون بدم میاد و به نظرم حیف وقت آدمیزاد که پای اینها صرف بشه. فکر می‌کنم، در مقابل، فیلم‌های انگلیسی‌زبان حتی اگر محتوی خیلی مناسبی نداشته باشند، باز هم برای تقویت زبان مناسب هستند. 

با این حال این فیلم رو استثنائا دوست داشتم. من دوست دارم که بعد از دیدن فیلم سرم درد بگیره و فکرم تا مدتی درگیر باشه که برای این فیلم همچین اتفاقی افتاد. 

پی‌نوشت: گزارش توسعه مهارت‌های فردی این ماه تو رو هم خواندم و لذت بردم. حلقه گم‌شده من تو این گزارش‌ها نسبت به تو، ورزش و پیاده‌روی منظم هست.

پی‌نوشت دو: راجع به تعادل چیزی ننوشتم چون نظر شخصی‌ای ندارم ولی به نظرم مطالبی که محمدرضا شعبانعلی راجع به «تعادل» در وبلاگش نوشته بسیار قابل تامل هست. 


با مهر
امیر

**Comment by masoudnasrollahi** [comment]
سلام امیرآقا، وقت بخیر.
«اگر خواستید گوش بدید...» کجا باید گوش بدیم؟ من لینک دسترسی به [«تاثیرگذارترین فایل»](https://aprd.ir/personal-development-farvardin-1402/#%d8%a8%d8%ad%d8%b1%d8%a7%d9%86-%d8%aa%da%a9%d8%b1%d8%a7%d8%b1-%d9%be%d8%b0%db%8c%d8%b1%db%8c-%d8%af%d8%b1-%d8%b9%d9%84%d9%85:~:text=%D8%A7%D9%86%D8%B5%D8%A7%D9%81%D8%A7%20%DB%8C%DA%A9%DB%8C%20%D8%A7%D8%B2-,%D8%AA%D8%A7%D8%AB%DB%8C%D8%B1%DA%AF%D8%B0%D8%A7%D8%B1%D8%AA%D8%B1%DB%8C%D9%86%20%D9%81%D8%A7%DB%8C%D9%84,-%D9%87%D8%A7%DB%8C%DB%8C%20%D8%A8%D9%88%D8%AF%20%DA%A9%D9%87) را نمی‌بینم.

عید امسال در مسیر رفت‌وبرگشت دیدوبازدیدها به فایل صوتی [«مرور چند نکته برای روزهای آغاز سال»](https://mrshabanali.com/%da%86%d9%86%d8%af-%d9%86%da%a9%d8%aa%d9%87-%d8%a8%d8%b1%d8%a7%db%8c-%d8%b1%d9%88%d8%b2%d9%87%d8%a7%db%8c-%d8%a2%d8%ba%d8%a7%d8%b2-%d8%b3%d8%a7%d9%84/) گوش دادم و این [عکس‌نوشته](https://s29.picofile.com/file/8463200642/1402_Theme.jpg) را برای خودم طراحی کردم. تو خود حدیث مفصل بخوان از این مجمل.

**Reply by pourmand1376** [reply]
سلام مسعود جان
با توجه به عکس نوشته‌ات فکر می‌کنم موضوع اصلی امسال تو با من خیلی اشتراک‌های زیادی داشته باشه. 

فایل رو می‌تونی از اینجا دریافت کنی. سه تا فایل هست که فایل اول و دوم برای من کافی بود. 

- [منابع و مدیریت منابع (Resource Management) | متمم](https://motamem.org/%D9%85%D8%AF%DB%8C%D8%B1%DB%8C%D8%AA-%D9%85%D9%86%D8%A7%D8%A8%D8%B9/)

**Reply by masoudnasrollahi** [reply]
این فایل‌ها داخل سری تفکر استراتژیک ارائه شده‌اند، دچار این کمال‌طلبی نمیشی که تا وقتی مطالب و درس‌های قبلی (پیش‌نیازش) را نخوانده‌ای، نیای سراغ این فایل‌ها؟

**Reply by pourmand1376** [reply]
سلام مسعود جان. راستش نه. مبارزه با کمال‌طلبی رو از خیلی وقت پیش شروع کردم و تا حد زیادی توش موفق شدم. 

این فایل‌ها رو هم از رادیو متمم پیدا کردم و دانلود کردم. وگرنه فایل‌های درس‌های جلوتر رو کمتر باز می‌کنم.

**Reply by masoudnasrollahi** [reply]
سلام آقا امیر 🙂 مرسی. چه زود جواب دادید. من گذاشتم ویندوز آپدیت بشه، بعد بیام نظرم را ویرایش کنم!
من خیلی با این ویرایشگر گیت‌هاب حال می‌کنم. دارم قابلیت‌ها و امکانات مختلفش را چک (و با کمترین احساس نیاز، اینجا، تست) می‌کنم. الان فهمیدم میشه عکس هم الصاق کرد، با یه drag and drop ساده یا ctrl+v.

- از چه شمشیری برای مبارزه استفاده کردید؟

من [عکس‌نوشته‌ام](https://github.com/pourmand1376/hugo-papermod-farsi-template/assets/77227411/e35dba42-bf38-4184-a1c2-70364654e0eb) را به‌روز کردم. احساسی که خیلی وقته داشته‌ام و متوجهش نبوده‌ام، اضطراب و دلشوره بوده. تب‌هایی چند ماه است روی مرورگر باز است، همۀ کتاب‌هایی که نصفه رها کرده‌ام، وبلاگ‌هایی که دوست دارم بخوانم، همه و همه این اضطراب را هر روز در دلم جای داده است. مسبب قسمی از اضطراب همین کمال‌طلبی‌ست و بقیه‌اش، دلایلی برای داشتن این تم. ببین چند وقت هست که من می‌خواستم این کامنت را ثبت کنم.
حالا نم‌نمک پیش می‌روم. پیروی و اعمال این تم روی جریان زندگی‌ام، دست‌کاری خیلی چیزها را می‌طلبد. خدا باعث‌وبانی این تم‌گذاری را بیامرزد.
([+](https://github.com/pourmand1376/hugo-papermod-farsi-template/discussions/12#discussioncomment-5873218))

**Reply by pourmand1376** [reply]
چه کار قشنگی کردی. من از امسال شروع کرده بودم برای سال یک سری هدف می‌گذاشتم که آخر سال بهشون برسم. ولی این شکل عکس‌نوشته درست کردن ایده بهتری هست. باعث میشه که آدم تعهد بیشتری داشته باشه. اصلاً هدف این گزارش‌های ماهانه هم همین تعهد بیشتر هست. همین که بصورت ناخودآگاه بدونم که آخر ماه باید مهارتی رو توسعه داده باشم که حرفی برای گفتن داشته باشم.
